client/state/ui/editor/post/actions.js

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
@mtias @rralian : This was also my concern.
I think we need to keep it 1 file in order to achieve "overview" of all post editor actions.
We already have `state/ui/editor/media` and `state/ui/editor/contactform` so I think this makes sense for now.
Notice that I am not implementing any reducers nor selectors right now. And I think actions can be split up in the future.
But I will reiterate that we need to gather them all in 1 place before we start to shove them in proper places.